theano/sandbox/cuda/tests/test_blas.py

```diff
@@ -90,22 +90,32 @@ def cmp(a_shp, b_shp):
                 [a, b],
                 tensor.dot(a, b) * numpy.asarray(4, 'float32'))
         t = f.maker.fgraph.toposort()
-        assert len(t) == 4, t
-        assert isinstance(t[0].op, tcn.GpuFromHost)
-        assert isinstance(t[1].op, tcn.GpuFromHost)
-        assert isinstance(t[2].op, tcn.blas.GpuDot22Scalar)
-        assert isinstance(t[3].op, tcn.HostFromGpu)
+        if any([isinstance(n.op, tcn.blas.GpuDot22Scalar) for n in t]):
```

Comment:
In which conditions should this happen? I thought `GpuDot22Scalar` was not used anymore.